code/modules/plumbing/plumbers/iv_drip.dm

```diff
@@ -7,11 +7,11 @@
 	density = TRUE
 	use_internal_storage = TRUE
 
-/obj/machinery/iv_drip/plumbing/Initialize(mapload)
+/obj/machinery/iv_drip/plumbing/Initialize(mapload, bolt=anchored, layer)
```

Comment:
Did you test this? I’m on my phone right now but I don’t think these are passed to non /obj/machinery/plumbing equipment

Reply:
Yes, I tested it. They're being passed regardless of the type of machinery, but `iv_drip/plumbing` just didn't take any other arguments before and ignored it. It works.

Reply:
Huh, not great but not your fault